scripts/calibration/jaiabot-calibration.py

```diff
@@ -326,15 +325,15 @@ def WriteBounds():
         while s is True:
             calibrate = input("do you want to calibrate the port elevator? Yes/No ")
             calibrate = calibrate.upper()
-            if calibrate == "YES":
+            if calibrate == "YES":9
```

Comment:
Should we be using magic numbers for this? Is there a variable we can put here instead?